.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+test.md
```

Comment:
- [ ] Great job adding the test.md file to the `gitignore`, :+1: But according to the project requirement, if you added it correctly, the file should not be pushed to GitHub, therefore it should not be present in this branch, please you should look into this. :smile: